### PR TITLE
ci(netlify): Configure netlify deploys

### PR DESCRIPTION
**Pull request by ff6347**
This PR adds a netlify.toml file for configuring the deploys on Netlify. @ZenVega Please check if this is right.

- The redirects for SPAs
- the build command
- the folder to deploy

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# all contexts unless otherwise overridden by more specific contexts.
+[build]
+
+  # Directory that contains the deploy-ready HTML files and
+  # assets generated by the build. This is relative to the base
+  # directory if one has been set, or the root directory if
+  # a base has not been set. This sample publishes the directory
+  # located at the absolute path "root/project/build-output"
+
+  publish = "dist/"
```
Comment (ZenVega) [inline netlify.toml]: ich glaube der ordner heisst "build". was eigentlich etwas ungünstig ist. vielelciht sollte man ihn noch auf dist ändern

Comment (ff6347) [reply netlify.toml]: I would keep it as CRA defines it. Changed it to build

**Comment by ZenVega** [comment]
ich glaube es gibt einen build error, weil die fonts im gitignore liegen und händisch eingefügt werden müssen: 
image

**Comment by ff6347** [comment]
> ich glaube es gibt einen build error, weil die fonts im gitignore liegen und händisch eingefügt werden müssen: image

This is suboptimal. If you want to use CI/CD you need to include those files in the repo or you need to have a build step that pulls these files in before you do the build on the CI server. Please take a look into the other repos of the org to see how this is handled elsewhere. I did a quick search and the Clan is used in the org but not the ClanWebPro.

**Comment by ZenVega** [comment]
I had this topic before with @vogelino . The idea was to make the font not public, since it's not a free font and therefore took it out to make it only available to our developers. Is there a better solution or should I just take it in again?

**Comment by MRigal** [comment]
@ZenVega ich glaube du kannst es fixen: https://app.netlify.com/sites/jahresbericht-2021/deploys/628508b5e463820008e56d5e#L116

@ff6347 würde der linter das nicht schon vorab abgreifen?

```
4:55:50 PM: Failed to compile.
4:55:50 PM: 
4:55:50 PM: src/components/ScrollableImages/index.js
4:55:50 PM:   Line 14:38:  'ui' is defined but never used  no-unused-vars
```

**Comment by ff6347** [comment]
> @ff6347 würde der linter das nicht schon vorab abgreifen?

If eslint is configured right it should warn in VSCode

**Comment by ZenVega** [comment]
yes, it warned me. wasn't an error though. should work now

**Review by ZenVega — APPROVED** [review]

**Comment by ff6347** [comment]
@all-contributors please add @ff6347 for infra

**Pull request merged**